### ISSUE TITLE
Add server tests to travis

### ISSUE THREAD

**Issue by averagehat** [DCGenomics/seqr#32]
The logic to start server (running `make`) could happen in `.travis.yml` or within a robot test.

This will add some time to the tests as it takes  some time to download solr, I don't think that we will hit memory limits on travis.

Related to #17

**Comment by lianyi** [comment]
That would be really cool to include the test in at least one of them, perhaps robot test first if that's easier.  It might make a lot of server side testing easier if we later want to increase the converge of testing, or adding new features on solr side.

**Comment by averagehat** [comment]
Currently it looks like the the download takes a long time, althought he problem may be in the script. We can get around Travis choking the build if it's longer than 10 minutes, although I'd rather have our builds be shorter. 
https://travis-ci.org/NCBI-Hackathons/seqr/builds/78597840

**Comment by lianyi** [comment]
Right, the downloading of whole solr dist might be a waste of time and bandwidth for every build. Maybe we can skip the server test in travis for now?  The "robot" seems enough to cover those tests.

**Comment by averagehat** [comment]
I think the way to approach this is to include the solr distribution in our repo

**Comment by lianyi** [comment]
Was the solr dist download the blocking issue?

-----Original Message-----

**Comment by averagehat** [comment]
Yes. Maybe hosting on S3 would be faster somehow too but putting it on the
repo is fine for now I think
